__tests__/InvalidMutations.ts

```diff
@@ -43,23 +43,33 @@ class R1 extends Reactor {
         this.addMutation(
             new Triggers(this.in1),
             new Args(this.in1, this.out2),
-            function(this, __in, __out) {
-                // test('expect error to be thrown on mutation creating loop', () => { 
-                //     expect(() => {
-                //         this.connect(__out, __in)
-                //     }).toThrowError("ERROR connecting " + __out + " to " + __in)
-                // })
+            function(this, __in1, __out2) {
+                test('expect error to be thrown on mutation creating loop', () => { 
+                    expect(() => {
+                        this.connect(__out2, __in1)
+                    }).toThrowError("New connection introduces cycle.")
+                    
+                    expect(() => {
+                        this.connect(__in1, __out2)
+                    }).toThrowError("New connection introduces direct feed through.") // dist port already in use                
```

Comment:
I believe that this test is wrong (even though it passes). Since there already is a mutation that establishes direct feed through, it doesn't seem like this connection should introduce it. I think the expected behavior here would be that the connection would be successful.